packages/pouchdb-binary-utils/src/readAsArrayBuffer.js

```diff
@@ -1,8 +1,17 @@
-// In Node.js, just convert the Buffer to a Buffer rather than
-// convert a Blob to an ArrayBuffer. This function is just a convenience
-// function so we can easily switch Node vs browser environments.
-function readAsArrayBuffer(buffer, callback) {
-  callback(buffer);
```

Comment:
it was useless to have a different node/browser version of this, because we never used the node one